group_vars/all.yml

```diff
@@ -0,0 +1,70 @@
+---
+# vars file for eessi project
+
+#cvmfs_geo_license_key: 
+
+# Automatically configure EESSI CVMFS repos
+eessi_cvmfs_repos_enabled: config-repo
+
+# Defaults for eessi-hpc.org config repo, syntax for each key is the same as that of cvmfs_<key | pluralize>
+eessi_cvmfs_config_repo:
+  domain: eessi-hpc.org
+  key:
+    path: /etc/cvmfs/keys/eessi-hpc.org/cvmfs-config.eessi-hpc.org.pub
+    key: |
+     -----BEGIN PUBLIC KEY-----
+     MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAqoTBc/Eksr9ECEOoCzHA
+     F20UjePR7mJyWB13eTBXnjq7PLKKilfk2llLKetPIRMEmbvp97NiQ9ALe/tiGDzT
+     1Wg82Mf9oO4DNQYrTfE/MP5inIrMtSv1WnYJYV4Bgm24s8IhHXytIf1D4qt0x1v5
+     0yRui0dwaZrErWKRKufbJ6ehmEDHuVSEAujrORyig/jYFKVyTaCsXnmwFDJmX2Uw
+     RGCaf0M9ZLYs9t/SWA0s08rle5kH82hgBsIvUr7PLT7hbU407VK6+Y02tuxbI0JR
+     +hCt494UsFs3tchmTebpK+x3pVRXybQ66Qm1mQiaz+2VbUJdQEov4RCC5PcOmOOk
+     BwIDAQAB
+     -----END PUBLIC KEY-----
+  urls:
+    - "http://cvmfs-s1-rug.eessi-hpc.org/cvmfs/@fqrn@"
+  repository:
+    repository: cvmfs-config.eessi-hpc.org
+    stratum0: cvmfs-s0.eessi-hpc.org
+    owner: "{{ cvmfs_repo_owner | default('root') }}"
+    server_options: []
+    client_options: []
+
+# Defaults for eessi-hpc.org repos
+eessi_cvmfs_keys:
+  - path: /etc/cvmfs/keys/eessi-hpc.org/pilot.eessi-hpc.org.pub
+    key: |     
+     -----BEGIN PUBLIC KEY-----
+     MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAommtaoqQ36Rb/cFnqUWg
+     8FoYCOjJWVBMQ4sVZ87oFRWehOUbcfy/ah85D9055HQws7i/hLNGJJAa7Es3E2o8
+     v8LtSlRdfR1A7b7LA6p3213HHz6dNU6I1riwtE2OP7/w4Q3+Agj1iRtW535GwzGo
+     4ZbewL8T1kbyaEm4HyefoitHXJthoPyTlYAhvAegIil1VJPhJZW+q/NWxnsd9QGJ
+     uAtbSICQg6mBJI50KiCBYmKHOAjH2W8qEnLWc7JoPoNvAHuMx1ya6YerBM5T9Esm
+     HJN0HHOVdRIwNjjPkV2jIqhJoI85no89v8/V45SFPVo+A7N0Cj4QdQrQTezpnlby
+     FQIDAQAB
+     -----END PUBLIC KEY-----
+
+cvmfs_cvmfs_server_urls:
+  - domain: eessi-hpc.org
+    use_geoapi: false
+    urls:
+      - "http://cvmfs-s1-rug.eessi-hpc.org/cvmfs/@fqrn@"
+      
+eessi_cvmfs_repositories:
+  - repository: pilot.eessi-hpc.org
+    stratum0: cvmfs-s0.eessi-hpc.org
+    owner: "{{ cvmfs_repo_owner | default('root') }}"
+    key_dir: /etc/cvmfs/keys/eessi-hpc.org
+    server_options:
+      - CVMFS_AUTO_GC=false
+      - CVMFS_GARBAGE_COLLECTION=true
+      - CVMFS_AUTO_GC=false
+    client_options:
+      - CVMFS_NFILES=4096
+
+# Set all the Galaxy defaults to EESSI defaults
```

Comment:
Maybe add an additional comment to explain why this is required?

I assume the `galaxy_cvmfs_*` variables are used somewhere else?

Reply:
Yes, they're used in the Ansible task files, so we really need at least some of them. Alternatively, we would have to change these variable names in all those files, which would make it much harder to pull in updates from the Galaxy repo.

Reply:
But I added a comment to explain this for now, and fixed a typo.